### ISSUE TITLE
Recommendations for all KB Articles

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10429]

**Comment by stefaniefgray** [comment]
YES to more images! Sadly, they can only be featured at the bottom of articles, not in between steps. Is there a way we can change this within Drupal?

**Comment by laflannery** [comment]
I have seen some articles with screenshots in the middle, like [this one](https://prod.cms.va.gov/help/vamc/about-locations-content-for-vamcs/how-to-add-other-va-locations).  I have literally no idea how any of these are built but based on some I've seen, I think somehow they got around this limitation?
